packages/common/src/auditlogs/AuditLogs.stories.tsx

```diff
@@ -0,0 +1,128 @@
+// Copyright 2024 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//@ts-nocheck
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+
+import { LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+
+import { getConfiguredStore } from '@northern.tech/store/store';
+import type { Meta, StoryObj } from '@storybook/react';
+import { HttpResponse, http } from 'msw';
+
+import { defaultState } from '../../../../tests/mockData';
+import AuditLogs from './AuditLogs';
+
+const meta: Meta<typeof AuditLogs> = {
+  component: AuditLogs,
+  argTypes: {
+    numberOfAuditLogs: {
+      control: { type: 'number', max: 1000, min: 0 },
+      name: 'Number of audit logs available'
+    }
+  }
+};
+
+export default meta;
+
+const preloadedState = {
+  ...defaultState,
+  app: {
+    ...defaultState.app,
+    features: {
+      ...defaultState.app.features,
+      hasAuditlogs: true,
+      isEnterprise: true
+    }
+  }
+};
+
+let logsNumber;
```

Comment:
An alternative to the global variable approach that I came across was to split the response size up across multiple stories with different entries returned - not quite as flexible, but might not feel as bad as a global 😅

Reply:
split it to main -105 logs and no logs. actually entering logs count have not that much benefits. I also tried decorators which supports context and I reinitialized parameters.msw there, but it took an effect only on the next rendering, so it looked laggy =)